src/en/send.md

```diff
@@ -353,3 +341,145 @@ If the request is not successful, the response body is `json`, refer to the tabl
+      },
+      "id":"0ea216ae-4b03-46b7-ab44-893ae85104f5",
+      "job_status":"pending",
+      "notification_count":3,
+      "original_file_name":"My bulk name",
+      "processing_finished":null,
+      "processing_started":null,
+      "scheduled_for":null,
+      "sender_id":null,
+      "service":"afa2be3b-1250-430f-a70f-28a1a9d49dfa",
+      "service_name":{
+         "name":"Test service"
+      },
+      "template":"659a214f-dfec-4882-9242-fea0bd502a09",
+      "template_version":4,
+      "updated_at":null
+   }
+}
+```
+
+You can follow the progression of your batch of notifications from the GC Notify web interface.
+
+If you scheduled your batch in the future, you can cancel it from the web interface.
+
+### Error codes
+
+If the request is not successful, the response body is `json`, refer to the table below for details.
+
+|status_code|message|How to fix|
+|:---|:---|:---|
+|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "You should specify either rows or csv"`<br>`}]`|Pass data through `rows` or `csv`|
+|`400`|`[{`<br>`"error": "ValidationError",`<br>`"message": "name is a required property"`<br>`}]`|Specify the `name` property|
+|`400`|`[{`<br>`"error": "ValidationError",`<br>`"message": "scheduled_for 42 is not of type string, null"`<br>`}]`|Check that you pass a valid ISO 8601 datetime|
+|`400`|`[{`<br>`"error": "ValidationError",`<br>`"message": "scheduled_for datetime cannot be in the past"`<br>`}]`|Check that you pass a datetime in the future|
+|`400`|`[{`<br>`"error": "ValidationError",`<br>`"message": "scheduled_for datetime can only be up to 96 hours in the future"`<br>`}]`|Check that you pass datetime at most 4 days in the future|
+|`400`|`[{`<br>`"error": "ValidationError",`<br>`"message": "scheduled_for datetime format is invalid. It must be a valid ISO8601 date time format, https://en.wikipedia.org/wiki/ISO_8601"`<br>`}]`|Check that you pass a valid ISO 8601 datetime|
+|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Template not found"`<br>`}]`|Update template ID|
+|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Template has been deleted"`<br>`}]`|Create a new template and update its ID|
+|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Service is not allowed to send emails"`<br>`}]`|Turn on email sending in settings|
+|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Missing column headers: name"`<br>`}]`|Add the missing column header|
+|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Duplicate column headers: name, NAME"`<br>`}]`|Remove the duplicate column headers|
+|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Too many rows. Maximum number of rows allowed is 50000"`<br>`}]`|Pass less than 50,000 rows|
+|`400`|`[{`<br>`{"error": "BadRequestError",`<br>`"message": "You cannot send to these recipients because you used a team and safelist API key."`<br>`}]`|Request to go live or use [a live API key](keys.md)|
```

Comment:
```suggestion
|`400`|`[{`<br>`{"error": "BadRequestError",`<br>`"message": "You cannot send to these recipients because you used a team and safelist API key."`<br>`}]`|Request to go live in Settings or use [a live API key](keys.md)|
```